node/src/chain_spec.rs

```diff
@@ -91,19 +92,19 @@ pub fn development_config() -> Result<ChainSpec, String> {
                 // SnowBridged Assets
                 vec![
                     (
-                        "Mangata".as_bytes().to_vec(),
-                        "MNG".as_bytes().to_vec(),
-                        "Mangata Asset".as_bytes().to_vec(),
+                        "Mangata".to_string().into_bytes(),
```

Comment:
Does `.to_string().into_bytes()` produce the same output as `.as_bytes().to_vec()`? If so, why `.to_string().into_bytes()` is preferred?